products/sle/main.pm

```diff
@@ -760,7 +760,7 @@ elsif (get_var("QA_TESTSET")) {
         loadtest 'kernel/kernel_kexec';
     }
     elsif (check_var('QA_TESTSET', 'kernel_multipath')) {
-        loadtest 'qa_automation/kernel_multipath';
+        loadtest 'kernel/multipath_iscsi';
```

Comment:
Schedule of this test and others will be handled in  https://progress.opensuse.org/issues/129113.